### DISCUSSION TITLE
RaspberryMatic unter Homeassistant OS / Supervised setzt alle Thermostate auf "manuell" bei Neustart

### DISCUSSION THREAD

**Discussion by tp1de** [OpenCCU/OpenCCU#2908] [🇩🇪 Fragen&Antworten] [answered]
Ich bin von der separaten Proxmox Installation auf die Home Assistant integrierte Version (auch unter Proxmox) umgezogen. 
Es funktioniert soweit alles.

Ärgerlich ist dabei nur, dass bei Neustart von RaspberryMatic oder HA alle Thermostate auf "manuell" mit 10 Grad Themperatur gesetzt werden.

Wie kann ich das verhindern?

**Comment by jens-maus** [comment, marked as answer]
Das hat nichts mit RaspberryMatic zu tun, sondern wird an den HmIP-WTH Gerätefirmwares liegen die du sicherlich vor kurzem auch hast installieren lassen. Bei diesen scheint es so zu sein, das wenn man nu die WTH stromlos macht und wieder Strom gibt diese standardmäßig auf MANUAL schalten. Ergo, an eQ3 melden.

Abgesehen davon solltest du aber übrigens - wenn du schon Proxmox nutzt - nicht dazu übergehen statt einer separaten RaspberryMatic VM stattdessen RaspberryMatic als Addon innerhalb von HA zu betreiben. Best practice ist und bleibt eine HA VM und eine RaspberryMatic VM parallel zu betreiben und diese via Homematic(IP) Local Integration zu koppeln.

**Comment by tp1de** [comment]
Vielen Dank für die schnelle Klarstellung. 
Die Thermostate (HmIP-eTRV bzw. HM-CC-RT-DN) werden aber nicht stromlos. Muss dann das "Umsetzen" auf manuell nicht von Raspberrymatic  oder der Homematic(IP) Local Integration kommen?

Ich werde wieder zurück auf die separate Proxmox VM gehen und schauen, ob es da auch auftritt.
Ich wollte die Komplexität etwas reduzieren und Services in HA integrieren (MQTT / MariaDB / Adguard und auch Raspberrymatic).

**Reply by jens-maus** [reply]
Wenn es kein stromlos machen ist/war, dann ist es einfach der Punkt das bei Neustart einer CCU zentrale die ja erst einmal abwarten muss bis sie von den Geräten den aktuellen Status frisch gemeldet bekommen hat. Bis dahin ist der Status (und eben auch Auto vs manual) ja undefiniert. Jedes HmIP Gerät meldet aber mind 1x pro stunde seinen Status. Also einfach abwarten und beobachten.

**Reply by jens-maus** [reply]
> Ich werde wieder zurück auf die separate Proxmox VM gehen und schauen, ob es da auch auftritt.
> 
> Ich wollte die Komplexität etwas reduzieren und Services in HA integrieren (MQTT / MariaDB / Adguard und auch Raspberrymatic).

Das ist aber eher der eierlegende Wollmilchsau Ansatz zu versuchen alles in eine VM zu questschen. Das ist erfahrungsgemäß auf lange Sicht der schlechtere Ansatz und besser ist es (wenn man schon Proxmox einsetzt) die Dienste zu separieren und jetzt nicht ne middleware damit aufzublähen auch noch Adguard machen zu müssen und damit als Router zu fungieren. Denn nicht alles was technisch machtbar ist ist auch technisch sinnvoll...

**Comment by Baxxy13** [comment]
Das...
![RM_STHD_nach_Neustart](https://github.com/user-attachments/assets/f8a9ec4c-c8dd-4d0c-9c06-9f0d7b1302e6)
... zeigt RM direkt nach Neustart z.B. für ein HmIP-STHD an.

Und das ist leider normal, aber das korrigiert sich wenn sich das Wandthermostat bei der Zentrale gemeldet hat.
Geändert wird bei einem Neustart der RM erstmal nichts am Thermostat, es sei denn du hast etwas programmiert.
Das trifft auch auf HA zu.

**Comment by tp1de** [comment]
Danke für das Feedback und die Klarstellungen. Obwohl ich RM bereits seit Jahren verwende, war mir das nie aufgefallen.
Es dauert bis zu einer Stunde, bis alle Thermostate sich bei der Zentrale gemeldet haben und die Werte stimmen.
